spec/redux/actionCreators-test.js

```diff
@@ -6,137 +6,104 @@ import thunk from 'redux-thunk';
 import Immutable from 'immutable';
 
 describe("The actionCreators'", () => {
-
   const middlewares = [thunk];
   const mockStore = configureStore(middlewares);
+  var store;
+
+  beforeEach(() => {
+    store = mockStore({});
+  });
+
+  describe("configureIDE function", () => {
+    var resolve, reject;
 
-  describe("configureIDE", () => {
-    var store;
     beforeEach(() => {
-      var runtimeApiLoader = () => new Promise(function(resolve, reject) {});
-      store = mockStore({});
+      var runtimeApiLoader = () => new Promise(function(_resolve, _reject) {
+        resolve = _resolve;
+        reject = _reject;
+      });
```

Comment:
I changed this to reflect how we initialize the runtimeApiLoader, as well as all of our promises in runCode, although technically this worked before